homeassistant/components/tahoma/sensor.py

```diff
@@ -86,6 +93,23 @@ def update(self):
                 float(self.tahoma_device.active_states["core:TemperatureState"]), 1
             )
             self._available = True
+        if (
+            self.tahoma_device.type
+            == "somfythermostat:SomfyThermostatTemperatureSensor"
+        ):
+            self.current_value = float(
+                "{:.2f}".format(
```

Comment:
Please use f-strings instead of str.format.

Reply:
sorry, I'm not very familiar with modern python standards... change done!